pyproject.toml

```diff
@@ -1,20 +1,30 @@
-[tool.black]
-line-length = 79
-exclude = '''
-(
-  /(  # generated files
-    .tox|
-    venv|
-    venv.*|
-    .venv.*|
-    target.*|
-    .*/build/lib/.*|
-    exporter/opentelemetry-exporter-zipkin-proto-http/src/opentelemetry/exporter/zipkin/proto/http/v2/gen|
-    opentelemetry-proto/src/opentelemetry/proto/.*/.*|
-    scripts
-  )/
-)
-'''
 [tool.pytest.ini_options]
 addopts = "-rs -v"
 log_cli = true
+
+[tool.ruff]
+# Minimum supported python version
+target-version = "py38"
+
+line-length = 79
+extend-exclude = [
+  # Ignore generated files
```

Comment:
Should we also ignore the previous files like `.tox` and `.venv`?

Reply:
It does by default https://docs.astral.sh/ruff/settings/#exclude

Reply:
Ruff's default includes these https://docs.astral.sh/ruff/configuration/#configuring-ruff. It also respects `.gitignore` files by default.

If anyone runs into any missing dirs for your local environment, we can ofc update